src/pipeline.ml

```diff
@@ -41,7 +41,7 @@ module Cluster = struct
   module Ci6_docker = Current_docker.Make(struct let docker_context = Some "docsci" end)
   module Toxis_docker = Current_docker.Make(struct let docker_context = Some "toxis" end)
   module Tezos_docker = Current_docker.Make(struct let docker_context = Some "tezos" end)
-  module Cb_docker = Current_docker.Make(struct let docker_context = Some "packet-current-bench" end)
+  module Cb_docker = Current_docker.Make(struct let docker_context = Some "packet_current_bench" end)
```

Comment:
This should be `packet-current-bench` in my opinion.That's what the docker context name is on the machine.

Reply:
That's right it should be the docker context. At some point they didn't line up properly.